### ISSUE TITLE
Multi-task concurrency function is required, which is very important for API users. Similar to ollama, this function has been implemented

### ISSUE THREAD

**Issue by windkwbs** [LostRuins/koboldcpp#866]
Multi-task concurrency function is required, which is very important for API users. Similar to ollama function has been implemented

**Comment by LostRuins** [comment]
Use `--multiuser` flag

**Comment by windkwbs** [comment]
--multiuser After checking, only queue execution is allowed and multiple tasks cannot be performed simultaneously

**Comment by LostRuins** [comment]
Yes, they will be executed in sequence. Koboldcpp does not allow parallel decoding.

**Comment by elter-tef** [comment]
Do you have plans to support parallel decoding in the future?

**Comment by LostRuins** [comment]
Not at this time
